xml/decoders/Zimo_Unified_software_v32_MX695_696_697V.xml

```diff
@@ -154,6 +156,7 @@
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV1-CV99.xml"/>
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV100-CV152version30.xml"/>
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV152bit7version27.xml"/>
+      <xi:include href="http://jmri.org/xml/decoders/zimo/CV152-CV157version28.xml"/>      
```

Comment:
You can edit the file within the PR by clicking the pencil icon while viewing the changes to the file. As long as you leave "Commit directly to the patch-2 branch." selected, when you commit the changes, the PR will automatically pick them up and run the tests again.